### DISCUSSION TITLE
Managing Large Experiments

### DISCUSSION THREAD

**Discussion by kramerrs** [satijalab/seurat#5387] [Q&A]
My group is struggling with scaling scRNA-seq projects.  One big problem is that collaborators inevitably want to view all expressed genes, or at least any given gene often times filtered out as low variation.  However some genes are just naturally not highly expressed, however when they are there it is very important.  The scale data slot is prohibitive.

Is there a way to use the data structure generated via SCTransform (without populating the ScaleData slot), to then generate scaled data for either one or some small subset of genes.  I don't know the algorithm completely, but it seems like this could be done from just the regression coefficients.

**Comment by saketkc** [comment]
You could use the `data` slot for visualizing expression - these are counts after regressing out the effect of sequencing depth and will be geranted for all genes (unless they get filtered out based on any of the criteria in the SCTransform() call).
